app/controllers/tasks_controller.rb

```diff
@@ -3,10 +3,14 @@ class TasksController < ApplicationController
   before_action :get_task, only: [:show, :edit, :update, :destroy]
 
   def show
+    if @task.label_id
+      @label = Label.find_by(id: @task.label_id)
+    end
   end
 
   def new
     @task = @list.tasks.build
+    @labels = Label.all
```

Comment:
I see `@labels = Label.all`,
should we move it into a `before_action` callback?